docs/modules/ROOT/pages/python-client-getting-started.adoc

```diff
@@ -0,0 +1,209 @@
+
+== What You Will Learn
+
+{description}
+
+== Before You Begin
+
+* A text editor
+
+* Hazelcast Cluster. You can use any of them below. In this tutorial, local cluster will be used.
+
+** Local Hazelcast Cluster
+
+** https://hazelcast.com/products/viridian/[Hazelcast Viridian] (Free account is available.)
+
+* Python 3.6+
+
+* Windows, Linux/UNIX or Mac OS X
+
+== Installing Hazelcast Cluster
+
+1. Go to https://hazelcast.com/open-source-projects/downloads/[Hazelcast Open Source Downloads] and download either the .zip or .tar distribution of Hazelcast.
+
+2. Decompress the contents into any directory that you want to run members from.
+
+3. Change into the directory that you decompressed the Hazelcast content and then into the bin directory.
+
+4. To start the cluster, use either hz-start or hz-start.bat depending on your operating system. Once you run the start script, you should see the Hazelcast logs in the terminal.
+
+== Installing and Starting with the Client
+
+Create a new folder by
+
+[source, bash]
+mkdir hazelcast-python-client-getting-started
+
+Navigate to that folder by
+
+[source, bash]
+cd hazelcast-python-client-getting-started
+
+Hazelcast Python Client can be installed by pip3. Run the following command.
+
+[source, bash]
+pip3 install hazelcast-python-client
+
+Create a file named tutorial.py
+
+Put the following code inside of tutorial.py.
+
+[source, python]
+----
+import hazelcast
+
+client = hazelcast.HazelcastClient()
+
+client.shutdown()
+----
+To run this Python script, use the following command in the terminal.
+
+[source, bash]
+python3 tutorial.py
+
+First line imports the Hazelcast Python client.
+
+Following line creates and starts a new Hazelcast Client. To configure the client, you can check https://hazelcast.readthedocs.io/en/stable/client.html#hazelcast.client.HazelcastClient[API documentation].
+
+== Use Map
+
+A Hazelcast map is a distributed key-value store, similar to Python Map structure. In the following example, we will work with map entries where the keys are session ids and the values are emails.
+
+[source, python]
+----
+import hazelcast
+
+client = hazelcast.HazelcastClient()
+
+some_map = client.get_map("some-map").blocking()
+
+some_map.set("12345", "example1@email.com")
+some_map.set("12346", "example2@email.com")
+
+entries = some_map.entry_set()
+
+for key, value in entries:
+    print(f"{key} -> {value}")
+
+client.shutdown()
+----
+
+
+The output snippet is
+[source, bash]
+----
+12346 -> example2@email.com
+12345 -> example1@email.com
+----
+
+Since Hazelcast clients work asynchronous, our output is not in order.
+
+Following line returns a map proxy object for the `some-map` named map.
+
+[source, python]
+some_map = client.get_map("some-map").blocking()
+
+If there is not a map named `some-map`, it will be automatically created. All the clients connected to the same cluster will have access to same map.
```

Comment:
```suggestion
If `some-map` doesn't exist, it will be automatically created. All the clients connected to the same cluster will have access to the same map.
```

Reply:
Done.